demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -1,21 +1,42 @@
 import os.path
+import re
+from typing import Optional, Tuple
 
-from demisto_sdk.commands.common.tools import (get_yaml, print_error,
+from demisto_sdk.commands.common.tools import (LOG_COLORS, get_yaml,
+                                               print_color, print_error,
                                                print_warning)
 from demisto_sdk.commands.generate_docs.common import (
     add_lines, build_example_dict, generate_numbered_section, generate_section,
     generate_table_section, save_output, stringEscapeMD)
 
 
+def append_or_replace_command_in_docs(old_docs: str, new_str: str, command_name: str) -> Tuple[str, list]:
+    regexp = rf'(?:###\s{command_name}).+?(?:(?=(?:\n###\s))|(?=(?:\n##\s))|\Z)'
```

Comment:
parts of this regex should be in consts so it will be readable what exactly this should do
see how its done in other places in the sdk

Reply:
I built that const file ><. just thought it's a one-time thing.